CHANGELOG.md

```diff
@@ -3,6 +3,24 @@
 ## v0.5.0 (WIP)
 
   * [components] ErrorTag: Renamed prop `phx_trigger_action` to `trigger_action`
+  * Introduce new notation for template and slot nodes (`<#template>` and `<#slot>` respectively)
+  * Add `<:slotname>` shorthand for `<#template slot="slotname">`
+  * Introduce block expressions for surface templates (`{#if}..{/if}`, `{#for}...{/for}`, etc.)
+  * Introduce `{#if}` block expression with support for `{#elseif}` and `{#else}` sub blocks
+  * Introduce `{#unless}` block expression
+  * Introduce `{#for}` block expression with support for `{#else}` sub block
+  * Introduce `s-` prefix as an alternative to `:` for directives (i.e. `s-if` and `:if` are now equivalent)
+  * Added a convert task to aid migrating to the new syntax
+
```

Comment:
Missing one line: Introduce `:values` directive for generating `phx-value-*` attributes